bedrock/firefox/redirects.py

```diff
@@ -291,7 +291,11 @@ def firefox_mobile_faq(request, *args, **kwargs):
              'https://support.mozilla.org/kb/about-tiles-new-tab'),
 
     # Bug 1239863
+    redirect(r'^firefox/os/1\.1/?$', 'firefox.os.index'),
+    redirect(r'^firefox/os/1\.3/?$', 'firefox.os.index'),
+    redirect(r'^firefox/os/1\.3t/?$', 'firefox.os.index'),
```

Comment:
This is why code-review is awesome. Totally forgot there was a 1.4 template to account for :wink: